tests/src/main/java/org/hibernate/beanvalidation/tck/tests/messageinterpolation/ExpressionLanguageMessageInterpolationTest.java

```diff
@@ -241,4 +253,33 @@ public boolean isValid(Continent continent, ConstraintValidatorContext constrain
 			}
 		}
 	}
+
+	private static class TestWrapper {
+
+		private final TestLogic testLogic;
+		private final Locale originalLocale;
+		private final Locale testLocale;
+
+		public TestWrapper(TestLogic testLogic, Locale testLocale) {
+			this.testLogic = testLogic;
+			this.originalLocale = Locale.getDefault();
+			this.testLocale = testLocale;
+		}
+
+		public void doTest() {
+			Locale.setDefault( testLocale );
+			testLogic.test();
+			Locale.setDefault( originalLocale );
+		}
+
+		public static void test(TestLogic testLogic) {
+			new TestWrapper( testLogic, Locale.ENGLISH ).doTest();
```

Comment:
@gsmet great, thanks for the update! I'll rework the last commit to ignore this import rules for this test